docs/docs/tutorial/chapter2/cells.md

```diff
@@ -28,14 +31,50 @@ export const Failure = ({ error }) => (
 
 export const Success = ({ posts }) => {
   return posts.map((post) => (
-    <article>
+    <article key={post.id}>
       <h2>{post.title}</h2>
       <div>{post.body}</div>
     </article>
   ))
 }
 ```
 
+</TabItem>
+<TabItem value="ts" label="TypeScript">
+
+```jsx
```

Comment:
```suggestion
```jsx
import type { FindPosts } from 'types/graphql' // 👈 generated type from `yarn rw g types`
import type { CellSuccessProps, CellFailureProps } from '@redwoodjs/web' // 👈 built in helper type for Cell props
```

Reply:
I've added the type imports, but didn't want to add the comments as those aren't actually generated. Did write it down on my own TODO list though to include similar notes in the text of the tutorial when I add the ability to have TS-only text